lib/facter/secure_boot_enabled.rb

```diff
@@ -0,0 +1,25 @@
+# _Description_
+#
+# Return true if system booted via uEFI Secure Boot
+#
+if Facter.value(:kernel).downcase == "linux" then
+  Facter.add("secure_boot_enabled") do
+    setcode do
+      if File.exist?('/sys/firmware/efi')
+        Dir.glob('/sys/firmware/efi/efivars/SecureBoot-*').each do | file |
+          File.open(file, 'r') do | hexcode |
+            hexcode.read(4)
+            code = hexcode.read(16).unpack('H*').first.to_i
+            if code == 1
```

Comment:
You can just leave this as `code == 1` and remove the `if/else` block since that will have the same result.

If you want to make this a bit cleaner, you could use a `retval`-style block where you set it to `false` by default and then set it to `code == 1` like in your #233 and #234 submissions.